cli/command/format.go

```diff
@@ -93,25 +102,41 @@ func genFormatPlaybook(curveadm *cli.CurveAdm,
 		return nil, errno.ERR_NO_DISK_FOR_FORMATTING
 	}
 
-	if options.showStatus && options.stopFormat {
+	showStatus := options.showStatus
+	stopFormat := options.stopFormat
+	debugFormat := options.debugFormat
+	if showStatus && stopFormat {
```

Comment:
Maybe you can do this in `PreRunE` phase, see:
https://github.com/opencurve/curveadm/blob/956c953249a5596ba300bb7541b8c69860284e52/cli/command/deploy.go#L139